### PR TITLE
Update README.md and docker-compose.yml

### PR DESCRIPTION
**Pull request by yashkohli88**
The README.md and docker-compose.yml file has been updated to fix the setup issues in development environment on MAC systems.

Platform flag has been added as "**linux/amd64**" to web, service and crawler services in docker-compose.yml file to maintain the compatibility. 

The MongoDB version should be downgraded to 4.4.28 from 5.0.6 in case mac systems do not have an AVX support. The step to downgrade the MongoDB version has been updated in the README.MD file.

### CODE REVIEW

**Review by qtomlinson — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -73,6 +73,14 @@ services:
       - "5000:5000"
 ```
 
+If you are using MAC systems and your CPU does not have an AVX support, please change the version of Mongo from 5.0.6 to 4.4.28 in docker-compose.yml file.
```
Comment (lumaxis) [inline README.md:76]:
```suggestion
If you are using a Mac computer and your CPU does not have a AVX support, please change the version of Mongo from 5.0.6 to 4.4.28 in docker-compose.yml file. 
```

File: docker-compose.yml [hunk 2]
```diff
@@ -47,8 +50,5 @@ services:
       - "27017:27017"
   clearlydefined_mongo_seed:
     image: "clearlydefined/docker_dev_env_experiment_clearlydefined_mongo_seed"
-    build:
-      context: ./mongo_seed
-      dockerfile: Dockerfile
```
Comment (lumaxis) [inline docker-compose.yml:52]: Why this change? 🤔

Comment (yashkohli88) [reply docker-compose.yml:52]: Building this clearlydefined_mongo_seed image through docker-compose on a windows system throws error as "**ERROR [clearlydefined_mongo_seed internal] load build context**". The usage of building this image was to add and test specific seed data and it is not necessary for the regular dev environment setup. Hence removed this section from the docker-compose.yml file so that there are no issues in setting up dev environment in any of the OS.

**Review by qtomlinson — APPROVED** [review]

File: docker-compose.yml (hunk 2 above)
Comment (elrayle) [reply docker-compose.yml:52]: Hi @yashkohli88.  At our dev meeting, I mentioned that it seems fine to remove the build if you add instructions in the README about how to run it manually.  Let me know when that is done and we'll get this merged.

Comment (qtomlinson) [reply docker-compose.yml:52]: @elrayle @yashkohli88 There is an existing [PR](https://github.com/clearlydefined/docker_dev_env_experiment/pull/12/files) documenting how to build and publish mongo_seed image.  Maybe we can add onto that PR to incorporate suggestions?

**Review by elrayle — APPROVED** [review]
This is great!  Thanks for making this easier to use.
